workflow/scripts/process/process_power_4_network.py

```diff
@@ -0,0 +1,255 @@
+"""
+Creates the network from the plants and targets data
```

Comment:
Outputting separate files for each dimension you're interested in may be absolutely the best way to do this, or it may be there are better approaches. We can discuss this in our meeting -- at the moment I'm not sure I understand the approach clearly enough to have an opinion :)